tenets/codelingo/psr2/elseif-not-else-if/codelingo.yaml

```diff
@@ -23,10 +30,7 @@ tenets:
       import codelingo/ast/php
 
       @review comment
-      # This does not work as intended.
-      # It removes everything after 'if'
-      # and before closing brace.
-      # Waiting on ability to edit raw file.
-      @rewrite --replace "elseif"
+      @rewrite --replace "{{ extractRaw(rawCode) }}"
       php.stmt_else(depth = any):
+        raw as rawCode
```

Comment:
Sorry, I don't quite understand. What does this do? It looks like you're replacing all of the spaces inside the else block. Such that this:

```go
else if {
   a := b + c
}
```

becomes:
```
elseif {
   a:=b+c
}
```

Reply:
@BlakeMScurr One of the quirks of JS. The replace method only replaces the first occurrence. So it only changes else if to elseif.